packages/app-mobile/components/screens/Note.tsx

```diff
@@ -189,6 +203,8 @@ class NoteScreenComponent extends BaseScreenComponent {
 								routeName: 'Note',
 								noteId: item.id,
 								noteHash: resourceUrlInfo.hash,
+								fromParentLinkNote: true,
+								parentLinkedNoteId: this.state.note.id,
```

Comment:
We have a `navHistory` stack. Why does this require a special case? And would that work when following multiple notes?